etcdutl/etcdutl/snapshot_command.go

```diff
@@ -48,6 +49,7 @@ func NewSnapshotCommand() *cobra.Command {
 	}
 	cmd.AddCommand(NewSnapshotRestoreCommand())
 	cmd.AddCommand(newSnapshotStatusCommand())
+	cmd.AddCommand(newSnapshotHashKVCommand())
```

Comment:
Probably it makes more sense to add a separate command something like `etcdutl hashkv <filename> [flags]`, because it isn't necessary related to a snapshot.